crates/sui-core/tests/staged/sui.yaml

```diff
@@ -500,12 +500,6 @@ ModuleId:
         TYPENAME: AccountAddress
     - name:
         TYPENAME: Identifier
-MoveFieldLayout:
```

Comment:
This feels possibly concerning? But I'm not really sure how it is used?
Maybe @bmwill, @mystenmark, or @lxfind would know?